pkg/security/events/event.go

```diff
@@ -13,13 +13,14 @@ import "github.com/DataDog/datadog-agent/pkg/security/secl/rules"
 // AgentContext serializes the agent context to JSON
 // easyjson:json
 type AgentContext struct {
-	RuleID        string `json:"rule_id"`
-	RuleVersion   string `json:"rule_version,omitempty"`
-	PolicyName    string `json:"policy_name,omitempty"`
-	PolicyVersion string `json:"policy_version,omitempty"`
-	Version       string `json:"version,omitempty"`
-	OS            string `json:"os,omitempty"`
-	Arch          string `json:"arch,omitempty"`
+	RuleID        string                   `json:"rule_id"`
+	RuleVersion   string                   `json:"rule_version,omitempty"`
+	RuleActions   []map[string]interface{} `json:"rule_actions,omitempty"`
```

Comment:
I'm not a fan of `map[string]interface{}`, why not using a real struct type ? This would simplify the `SendEvent` later down